src/generic/eigen_solver.h

```diff
@@ -217,37 +297,174 @@ namespace oomph
   };
 
 
+  ///////////////////////////////////////////////////////////////////////
+  ///////////////////////////////////////////////////////////////////////
+  ///////////////////////////////////////////////////////////////////////
+
   //=====================================================================
-  /// Class for the LAPACK eigensolver
+  /// Class for the LAPACK QZ eigensolver
   //=====================================================================
   class LAPACK_QZ : public EigenSolver
   {
   public:
     /// Empty constructor
-    LAPACK_QZ() : EigenSolver() {}
+    LAPACK_QZ() : EigenSolver(), Tolerance_for_ccness_check(1.0e-13) {}
```

Comment:
Given that the Tolerance is only used in PARANOID should this initialisation be PARANOIDed away?

Reply:
No because that would produce uninitialised data. Baaaaad!.We could paranoid away the definition but that's always a bit scary when people mix and match versions that were compiled with different settings so I'd rather not.

Done.

Reply:
OK. Fair point.